maps/biter_battles_v2/feeding.lua

```diff
@@ -233,6 +233,22 @@ function set_evo_and_threat(flask_amount, food, biter_force_name)
 		threat = threat * (100 / (100.001 - reanim_chance))
 	end
 	global.bb_threat[biter_force_name] = math_round(global.bb_threat[biter_force_name] + threat, decimals)
+	
+	if global.active_special_games["shared_science_throw"] then
+		local enemyBitersForceName = ""
+		if biter_force_name == "south_biters" then
+			enemyBitersForceName = "north_biters"
+		else
+			enemyBitersForceName = "south_biters"
+		end
```

Comment:
It can be done with `Tables` (but doesn't have to)
```suggestion
		local enemyBitersForceName = tables.enemy_team_of[tables.force_translation[biter_force_name]] .. "_biters"
```

Reply:
Yeah it works as is, but it's cleaner your way and much shorter code which means easier to maintain, so I committed it, tested it, nothing breaking with this change.